aapcs64/aapcs64.rst

```diff
@@ -892,6 +892,11 @@ thread-local storage on platforms where multi-threaded code is
 supported.  The exact location of such information is platform
 specific.
 
+The FPMR is system register that controls behaviors of the FP8 instructions. It
+is a temporary register. The state of the register must be held in thread-local
+storage on platforms where multi-threaded code is supported. The exact location
+of that storage is platform specific.
```

Comment:
```suggestion
is a temporary register.
```
I don't think we want the part about thread-local storage, for two reasons:

*   The idea of storing state in thread-local storage seems inconsistent with having a temporary/caller-saved register.  (This is contrast to something like FPCR, which is maintained cooperatively by the thread, rather than being caller-saved or callee-saved.)

*   The point of the corresponding decimal floating-point wording is that the DFP state is emulated rather than architected.  That is, DFP relies on something like the FPCR, but since there is no architected support for DFP, there is no architected support for a DFP-like FPCR either.  The DFP emulation code must therefore emulate both the DFP operations themselves and the thread-local nature of the control bits.

Reply:
I'm open to rewording, perhaps not using the overloaded term "thread-local storage", but anyway IMO we would like to state exlicitly that the register contains thread-private data and must be saved/restored durign context switch, whuch  might be obviuos for temporary GPRs, but not for a system register.

Reply:
Done